templates/zerver/reset_confirm.html

```diff
@@ -40,19 +40,16 @@ <h1>{{ _('Set a new password.') }}</h1>
                         <div class="alert alert-error">{{ error }}</div>
                         {% endfor %}
                     {% endif %}
-                </div>
-                <div class="input-box">
-                    <div class="">
-                        <div class="progress" id="pw_strength">
-                            <div class="bar bar-danger" style="width: 10%;"></div>
-                        </div>
+                    <div class="progress" id="pw_strength">
+                        <div class="bar bar-danger" style="width: 10%;"></div>
                     </div>
+                    <span id="reset-password-limit-indicator" class="tippy-zulip-tooltip" data-tippy-content="{{ ('Maximum password length: 100 characters') }}"></span>
```

Comment:
Don't think there is a need for two separate commits, all changes can be done in one commit, since these elements are eventually used in the second commit.